### DISCUSSION TITLE
Radeon VII - Gigabyte vs XFX

### DISCUSSION THREAD

**Discussion by valeriob01** [preda/gpuowl#221] [Show and tell]
I have both and there is a big difference: the Gigabyte produces always some errors during gpuowl run, there's nothing I can do about this, conversely the XFX runs smoothly without errors at all.

**Comment by preda** [comment]
It seems there is a difference between the Hynix memory (fully stable) and the Samsung memory (errorprone). Can you confirm the RAM manufacturer on your GPUs?

I have the same situation myself (3xHynix R7, 1xSamsung R7), and the same situation was reported on mersenneforum by Ken.

Also interesting, there was some discussion on ROCm 4.1 about ECC and R7.

**Comment by valeriob01** [comment]
I don't know yet how to check for the RAM manufacturer, do you know how to do this?

**Reply by preda** [reply]
```
$ ~/ROC-smi/rocm-smi --showmemvendor 


========================ROCm System Management Interface========================
================================================================================
GPU[0] 		: GPU memory vendor: samsung
GPU[1] 		: GPU memory vendor: hynix
GPU[2] 		: GPU memory vendor: hynix
GPU[3] 		: GPU memory vendor: hynix
================================================================================
==============================End of ROCm SMI Log ==============================
```

**Reply by valeriob01** [reply]
I must have some permission problem because on both computers it shows "Unable to get GPU memory vendor." ...

**Reply by valeriob01** [reply]
Attempted to correct permissions with MODE="0660" in /etc/udev/rules.d/70-kfd.rules and also adding myself to the "render" group without success.

**Reply by valeriob01** [reply]
Posted new issue at ROCm github https://github.com/RadeonOpenCompute/ROCm/issues/1434

**Reply by valeriob01** [reply]
After upgrading to ROCm 4.1.0 I can confirm, Gigabyte has samsung memory and XFX has hynix. This version of ROCm is just a little bit slower.

**Comment by valeriob01** [comment]
Posterity will judge on why Gigabyte uses Samsung memory while XFX uses Hynix, and on why Samsung memory is so unstable...
